pkg/portal/cluster/machines.go

```diff
@@ -5,11 +5,20 @@ package cluster
 
 import (
 	"context"
+	"strings"
 
+	mgmtcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"
 	machinev1beta1 "github.com/openshift/api/machine/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/compute"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/features"
 )
 
+// Creating local vars for these functions in order to make them testable. The function definition will is altered while in the tests written.
```

Comment:
The intent here is to not make them local(which they aren't anyway, as they are exported), but to be able to swap the function definitions or make them swappable during testing. So, consider changing the comment.

Reply:
So here .. in the comment, I have talked about creating local variables not making the functions local. Yes they are exported .. which is not required so I will make them local and also will change the comment to make it more descriptive to give an idea of why that is done.